image/manifest.go

```diff
@@ -89,7 +89,7 @@ func (m *manifest) validate(w walker) error {
 
 func (m *manifest) unpack(w walker, dest string) error {
 	for _, d := range m.Layers {
-		if d.MediaType != string(schema.MediaTypeImageConfig) {
+		if d.MediaType != string(schema.MediaTypeImageLayer) {
 			continue
```

Comment:
Although the spec [does not currently say that layers MUST have a particular type][1], so this is currently an:

> unimplemented: unpacking layer with media-type %q

I expect the spec should clarify if it places bounds (upper or lower) on manifest layer[] media types, and once that happens we can adjust this to match.

[1]: https://github.com/opencontainers/image-spec/blob/master/manifest.md#image-manifest-property-descriptions